src/performance/scala-2.11/markets/MarketActorBenchmarkSimulation.scala

```diff
@@ -0,0 +1,79 @@
+package markets
+
+import akka.actor.{ActorRef, ActorSystem, Props}
+import akka.agent.Agent
+import akka.testkit.TestKit
+
+import com.typesafe.config.{Config, ConfigFactory}
+import markets.engines.CDAMatchingEngine
+import markets.orders.orderings.ask.AskPriceTimeOrdering
+import markets.orders.orderings.bid.BidPriceTimeOrdering
+import markets.participants.strategies.TestRandomTradingStrategy
+import markets.participants.{SubmitAskOrder, SubmitBidOrder}
+import markets.settlement.TestSettlementMechanismActor
+import markets.tickers.Tick
+import markets.tradables.{TestTradable, Tradable}
+
+import scala.collection.mutable
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2424830928/issues/source?bid=2921361&fileBranchId=3239634#l17)